### PR TITLE
Fix allow empty postcode when going through /tenancy-start

### PR DESCRIPTION
**Pull request by sulthan-ahmed**
Previously when you entered a manual address the postcode was undefined, as a result the tenancy-start behaviour did not deal with that

### CODE REVIEW

File: apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js [hunk 1]
```diff
@@ -5,7 +5,12 @@ const pilotStartDate = new Date('2014/12/01').getTime();
 const pilotEndDate = new Date('2016/01/31').getTime();
 const _ = require('lodash');
 const inRange = date => date >= pilotStartDate && date <= pilotEndDate;
-const isPilot = postcode => _.some(pilotPostcodes, pilot => postcode.startsWith(pilot));
+const isPilotOrUndefined = postcode => {
+  if (postcode === undefined) {
+    return true;
+  }
```
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js]: I could have put the logic below but it just made the code more messy

File: apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js [hunk 2]
```diff
@@ -15,7 +20,7 @@ module.exports = superclass => class extends superclass {
       const postcode = req.sessionModel.get('property-address-postcode');
 
       if (inRange(tenancyStart)) {
-        req.sessionModel.set('valid-tenancy', isPilot(postcode));
+        req.sessionModel.set('valid-tenancy', isPilotOrUndefined(postcode));
```
Comment (sulthan-ahmed) [inline apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js]: As mentioned above I could have put the if statement here but it just would have been harder to read

File: test/unit/behaviours/tenancy-start-postcode-check.test.js [hunk 3]
```diff
@@ -106,8 +106,8 @@ describe('apps/behaviours/tenancy-start-postcode-check', () => {
         });
       });
 
-      it(`if the tenancyStart is between 2014-12-01 & 2016-01-31 &
-        the postcode is NOT in the West Midlands do NOT set valid-tenancy to true`, (done) => {
+      it(`the postcode is NOT in the West Midlands do NOT set valid-tenancy to
```
Comment (sulthan-ahmed) [inline test/unit/behaviours/tenancy-start-postcode-check.test.js]: the `between 2014...` is already in the describe above so this was duplicated

File: apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js (hunk 1 above)
Comment (JoeChapman) [reply apps/right-to-rent-check/behaviours/tenancy-start-postcode-check.js]: does this mean that if the postcode has not been set, then the tenancy start date is valid?

**Pull request merged**